test/upgrade/test-upgrade.bats

```diff
@@ -212,6 +209,25 @@ failed    | exited     | 17
     done < <(parse_table "$tests")
 }
 
+@test "network - curl" {
+    run curl --max-time 3 -s 127.0.0.1:$HOST_PORT/index.txt
+    is "$output" "$RANDOM_STRING_1" "curl on running container"
+    # restart the container and check if we can still use the port
+    run_podman stop -t0 myrunningcontainer
+    run_podman start myrunningcontainer
+    run curl --max-time 3 -s 127.0.0.1:$HOST_PORT/index.txt
+    is "$output" "$RANDOM_STRING_1" "curl on running container"
```

Comment:
Suggestion: `s/running/restarted/`. I adore unique error messages because if that's the only thing I see in a log, I can still find out the exact place in the code where the failure happened.